apps/metadata-editor/src/app/edit/components/publish-button/publish-button.component.html

```diff
@@ -11,12 +13,49 @@
     <mat-spinner [diameter]="16"></mat-spinner>
   </gn-ui-button>
   <gn-ui-button
+    cdkOverlayOrigin
+    #actionMenuButton
     *ngSwitchCase="'upToDate'"
     type="secondary"
     [disabled]="true"
-    (buttonClick)="saveRecord()"
+    (buttonClick)="publishRecord()"
     [matTooltip]="'editor.record.upToDate' | translate"
   >
     <ng-icon name="matCheckCircleOutline"></ng-icon>
   </gn-ui-button>
+  <ng-template #template>
+    <div
+      data-test="publish-warning"
+      class="w-96 p-5 flex flex-col gap-3 mt-2 border border-gray-100 bg-white shadow-2xl rounded-2xl mr-2"
+    >
+      <span
+        class="text-center"
+        translate
+        [translateParams]="{
+          date: publishWarning[0],
+          user: publishWarning[1]
```

Comment:
Not sure what has been discussed, just wondering if it could make sense to also display the given name and family name here?